conformance/README.md

```diff
@@ -607,33 +607,51 @@ expectation, ending that branch of the test tree.
 These rules describe Ion data-model results for use in the `denotes` expectation:
 
 ```ebnf
-model-value   ::=  model-content  |  annotated
-
-model-content ::=  null.null
-                |  bool
-                |  int
-                |  string
-                |  "("  "null"    model-type    ")"
-                |  "("  "string"  codepoint*    ")"
-                |  "("  "symbol"  model-symtok  ")"
-                |  "("  "list"    model-value*  ")"
-                |  "("  "sexp"    model-value*  ")"
-                |  "("  "struct"  model-field*  ")"
-                |  "("  "blob"    bytes*        ")"
-                |  "("  "clob"    bytes*        ")"
+model-value     ::=  model-content  |  annotated
+
+model-content   ::=  null.null
+                  |  bool
+                  |  int
+                  |  string
+                  |  "("  "null"      model-type       ")"
+                  |  "("  "bool"      bool             ")"
+                  |  "("  "int"       int              ")"
+                  |  "("  "float"     string           ")" // See https://amazon-ion.github.io/ion-docs/docs/float.html
```

Comment:
This feels like a cop-out, but I am not educated enough to know a better alternative.  Good enough for now, we can revisit later.

Reply:
It is for sure a cop out, but I don't think we really want to get into binary representations of floats. If we write it as an Ion or JSON number, then we could run into a problem where the text parser is just testing against itself, which means it cannot detect errors. By encoding it as a string, we can use things like `Double.valueOf(String)`, so that we can use the standard library as the reference for our parser implementations (which, I know, mostly just wrap standard libraries for this anyway).

Reply:
For floats, I suspect it will make most sense to just use `produces` and check that multiple text and binary representations produce the same value.
```
(ion_1_1 "reading non-numeric float values"
    (then (each "nan"
                (text "nan")
                (binary "6B 01 7C")
                (binary "6C 00 00 C0 7F")
                (binary "6D 01 00 00 00 00 00 F0 7F")
                // ...
                (produces nan)
    (then (each "positive infinity"
                // ...

```